pages/form-builder/contactus.tsx

```diff
@@ -0,0 +1,281 @@
+                        required: true,
+                      },
+                      {
+                        id: "request-demo",
+                        name: "demo",
+                        label: t("contactus.request.option3"),
+                        required: true,
+                      },
+                      {
+                        id: "request-other",
+                        name: "other",
+                        label: t("contactus.request.option4"),
+                        required: true,
+                      },
+                    ]}
+                  ></MultipleChoiceGroup>
+                </fieldset>
+                <div className="focus-group">
+                  <Label
+                    id={"label-description"}
+                    htmlFor={"description"}
+                    className="required"
+                    required
+                  >
+                    {t("contactus.description.title")}
+                  </Label>
+                  <Description id={"description-description"}>
+                    {t("contactus.description.description")}
+                  </Description>
+                  <TextArea
+                    id={"description"}
+                    name={"description"}
+                    className="required w-[34rem] mt-4"
+                    required
+                    characterCountMessages={{
+                      part1: t("formElements.characterCount.part1"),
+                      part2: t("formElements.characterCount.part2"),
+                      part1Error: t("formElements.characterCount.part1-error"),
+                      part2Error: t("formElements.characterCount.part2-error"),
+                    }}
+                  />
+                </div>
+                <div className="flex mt-14">
+                  <Button
+                    type="submit"
+                    className={` 
+                        mr-8
+                        bg-blue-dark text-white-default border-black-default py-4 px-8 rounded-lg border-2 border-solid
+                        hover:text-white-default hover:bg-blue-light active:text-white-default active:bg-blue-active active:top-0.5
+                        focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500
+                      `}
+                    disabled={submitting}
+                  >
+                    {t("submitButton", { ns: "common" })}
+                  </Button>
+                </div>
+              </form>
+            </>
+          )}
+        </Formik>
+      )}
+      {isSuccessScreen && (
+        <>
+          <h1>{t("requestSuccess.title")}</h1>
+          <p className="mb-16 mt-[-2rem] font-bold">{t("requestSuccess.paragraph1")}</p>
+          <div className="mb-16">
+            <StyledLink
+              href={`/${i18n.language}/myforms`}
+              className={`
```

Comment:
Perhaps use the button mentioned above and extend it such that it overrides the default link styles